thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2961,10 +2961,12 @@ void ProgressInfo::processInfo() // reimplement as counts have special flags (i.
     ForEachItemIn(n, counts)
     {
         unsigned __int64 thiscount = counts.item(n);
-        if (thiscount & THORDATALINK_STARTED)
-            startcount++;
         if (thiscount & THORDATALINK_STOPPED)
+        {
+            startcount++;
             stopcount++;
+        } else if (thiscount & THORDATALINK_STARTED)
```

Comment:
style: else on newline to be consistent with rest of class/file (and preferred style anyway).